MTA-6931 Release Notes for 8.1.3 (#400)
migtools/mta-documentation: docs/topics/release-notes-topics/ref_known-issues-8-1-3.adoc

:_newdoc-version: 2.18.7
:_template-generated: 2026-07-01
:_mod-docs-content-type: REFERENCE

[id="known-issues-8-1-3_{context}"]
= Known issues

[role="_abstract"]
{ProductFirstRef} version 8.1.3 is affected by the following newly identified and previously known issues. A known issue is listed in all future release notes until resolved, at which point it is published as a fixed issue.

{ProductShortName} extension renders inconsistently with Mozilla Firefox browser 149.0::

> The {ProductShortName} Visual Studio Code extension 8.1 renders inconsistently in the Red{nbsp}Hat OpenShift Dev Spaces dashboard when using Mozilla Firefox browser version 149.0.

Reviewer: ```suggestion
The {ProductShortName} Visual Studio Code extension 8.1 renders inconsistently in the Red{nbsp}Hat OpenShift Dev Spaces dashboard when you use the Mozilla Firefox browser version 149.0.
```

+

> To work around the problem, use the {ProductShortName} Visual Studio Code extension in Red{nbsp}Hat OpenShift Dev Spaces through the Google Chrome browser.

Reviewer: Just minor KI template fix.
```suggestion
To work around this problem, use the {ProductShortName} Visual Studio Code extension in Red{nbsp}Hat OpenShift Dev Spaces through the Google Chrome browser.
```

+
link:https://redhat.atlassian.net/browse/MTA-6735[MTA-6735]

Code review editor overwrites source code with unsaved code resolutions::
+

> After closing the review editor without saving a code resolution suggestion, {mta-dl-plugin} modifies the source code with the suggested changes. As a consequence, the source code state changes, yet the Git status does not show any state change for the file. To work around the problem, accept or reject the code changes in the Visual Studio Code editor.

Reviewer: Let's just have the workaround line as a separate one, same as in the previous KI. So just for consistency.
```suggestion
After closing the review editor without saving a code resolution suggestion, {mta-dl-plugin} modifies the source code with the suggested changes. As a consequence, the source code state changes, yet the Git status does not show any state change for the file.
+
To work around this problem, accept or reject the code changes in the Visual Studio Code editor.
```
